slave.mk

```diff
@@ -1186,6 +1186,8 @@ $(addprefix $(TARGET_PATH)/, $(DOCKER_IMAGES)) : $(TARGET_PATH)/%.gz : .platform
 			--build-arg SONIC_VERSION_CACHE=$(SONIC_VERSION_CACHE) \
 			--build-arg SONIC_VERSION_CACHE_SOURCE=$(SONIC_VERSION_CACHE_SOURCE) \
 			--build-arg image_version=$(SONIC_IMAGE_VERSION) \
+			--build-arg eventd_enabled=$(INCLUDE_SYSTEM_EVENTD) \
```

Comment:
>eventd_enabled [](http://example.com/codeflow?start=15&length=14)

let's keep using consistent words. eventd_enabled->include_eventd.

We normally use "include_" for build time flags.

Reply:
Changed